src/cgame/cg_rocket_datasource.cpp

```diff
@@ -547,6 +569,9 @@ void CG_Rocket_BuildResolutionList( const char* )
 
 	buf[ 0 ] = '\0';
 
+	// Sort resolutions by size by default (larger first).
+	qsort( rocketInfo.data.resolutions, rocketInfo.data.resolutionCount, sizeof( resolution_t ), &ResolutionListCmpBySize );
```

Comment:
Use std::sort. `std::sort(rocketInfo.data.resolutions,rocketInfo.data.resolutions+rocketInfo.data.resolutionCount,[](const resolution_t&a, const resolution_t& b){return a.width*a.height < b.width*b.height;});`

Reply:
Done (your suggestion listed smaller first, turned it to larger first).